src-docs/src/components/guide_components.scss

```
@@ -177,7 +179,7 @@ $elasticLogoTextDark: #1C1E23;
 }
 
 .guideDemo__ghostBackground {
-  @if (lightness($ouiTextColor) < 50) {
+  @if  lightness($ouiTextColor) < 50 {
```

Why the extra space here? There seem to be a lot of these? Is it just an artifact of `yarn lint-sass-fix`?

```suggestion
  @if lightness($ouiTextColor) < 50 {
```

---

It is artifact from `yarn lint-style --fix`. I was wondering to fix it with `prettier`. But we need to update `prettier` first and it is another task.
 
I tend to avoid manual intervention and keep it as is for now. Prettier would reformat it later